### ISSUE TITLE
Embedding javascript in the output HTML tmap file

### ISSUE THREAD

**Issue by aparente-nurix** [reymond-group/tmap#13]
When generating a tmap in an jupyter notebook, the output is a single HTML file that includes the javascript used to plot the map within the HTML file. Oddly, when turning the same code into a script, it generates the .js and .html files separately. Is there an option that can be called to embed the javascript within the html file, such that there's only one file thats created?

**Comment by Lerbytech** [comment]
As far as I know, js script can be embedded to html file. Thus, you can write simple script to merge two files together. I think that BeautifulSoup is good enough to help you with this mission

**Comment by aparente-nurix** [comment]
Yeah, it can totally be merged after the files are created. However, I’m wondering why the files automatically get merged in jupyter notebook but not when run in a script. Is there a flag that can be set in tmap to merge them automatically?
